src/main/java/fr/silverpricing/api/model/Chambre.java

```diff
@@ -1,8 +1,12 @@
 package fr.silverpricing.api.model;
```

Comment:
:warning: The createCoordinates method does not handle null pointer exceptions properly and throws a generic exception. (Severity: Major)